src/OSPSuite.SimModelNative/src/CppODEExporter.cpp

```diff
@@ -572,9 +585,13 @@ namespace SimModelNative
 		const std::map<int, formulaParameterInfo > & formulaParameterIDs, const std::string &jacConstants,
 		const std::vector<Observer*> &obs, ostream & os) {
 
-		// ODE initial values, dont use GetDEInitialValues, because some might be defined as a formula
```

Comment:
not sure why @hdiedam made this comment and implemented it as before - it's exactly the opposite; and the previous implementation crashed if initial values were given by non-simplifiable formulas